python/ml4ir/base/model/relevance_model.py

```diff
@@ -325,20 +346,22 @@ def evaluate(
         Args:
```

Comment:
line 323: "Evaluate the ranking model" it's actually the base model. And we would need to provide more info, because we do more than just evaluating it (returning metrics, saving layers, etc.)